src/Pages/Detail/Detail.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import API from '../../config';
+import Goods from './DetailSections/Goods';
+import Categories from './DetailSections/Categories';
+import Popup from '../../Components/ShoppingModal/Popup';
+import Login from '../Login/Login';
+import './Detail.scss';
+
+class Detail extends React.Component {
+  state = {
+    goods: {},
+    isModalOpen: false,
+    isLogin: false,
+    content: {},
+  };
+
+  handleModal = (e, amount) => {
+    const { name } = e.target;
```

Comment:
```suggestion
    const { name: buttonName } = e.target;
```
이런 식으로 특정 스코프 내에서 변수에 별칭을 줄 수 있습니다

Reply:
네 별칭으로 변경하겠습니다.